filter_primitive:
  move_group: "upper_body"
  links : "rh_palm"
    
